class-03-responsive-mobile-first/pair-assignment/starter-code/styles/modules.css

```diff
@@ -0,0 +1,100 @@
+.site-header {
+  background: #FFEAAA;
+}
+
+/* DONE: Float the main header to the left, to make room for the hamburger menu. */
+.heading-group-h1 {
+  float: left;
+  padding: 0 0 .2em 1em;
+}
+
+/* TOD: Add styles for the icon-menu class. It should be bigger, and off to the right. */
+.icon-menu {
+  position: absolute;
+  right: 4px;
+  top: 27px;
+  height: 2em;
+  width: 2em;
+  color: black;
+}
+
+.main-nav {
+  padding: 0.1em;
+}
+
+.main-nav ul {
+  display: none;
+  padding: 0;
+  list-style: none;
+  clear: both;
+}
+
+.main-nav:hover ul {
+  display: block;
+}
+
+.main-nav li {
+  margin-right: .4em;
+  font-size: 1.5em;
+  padding: .5em .2em;
+  /*border-bottom: 1px solid #ccc;*/
+}
+
+#filters {
+  padding: 0;
+  list-style: none;
+}
+
+.template {
+  display: none;
+}
+
+.byline {
+  color: #777;
+}
+
+.byline a {
+  font-weight: bold;
+  text-decoration: none;
+}
+
+/* TODO: Now add styles for how things should look on a larger screen:
+         - Hide the hamburger.
+         - Main should be all on one row, with padding spaced better for a larger screen.
+         - Just, ya know, try to make it look good! */
+
+@media screen and (min-width: 480px) {
+   .icon-menu {
+     visibility: hidden;
+   }
+
+   .main-nav ul  {
+     display: block;
+   }
+   nav li {
+     display: inline;
+   }
+   /*#home {
+     float: left;
+   }
+   #about {
+     float: left;
+     left: 22%;
+   }
+   #twitter {
+     float: left;
+     left: 44%;
+   }
+   #git {
+     float: left;
+     left: 66%;
+   }*/
+
+}
+
+@media not (min-width: 480px) {
+  .main-nav:hover ul {
+    display: block;
+  }
```

Comment:
Keeping with mobile first design principals this should be covered by the default CSS.